fem/bilininteg_vectorfe.cpp

```diff
@@ -372,8 +372,8 @@ void PAHcurlHdivSetup2D(const int Q1D,
                const double R21 = D2*J21;
                const double R22 = D2*J22;
                y(i11,qx,qy,e) = w_detJ * ( J22*R11 - J12*R21); // 1,1
-               y(i12,qx,qy,e) = w_detJ * ( J22*R12 - J12*R22); // 1,2
-               y(i21,qx,qy,e) = w_detJ * (-J21*R11 + J11*R21); // 2,1
+               y(i21,qx,qy,e) = w_detJ * ( J22*R12 - J12*R22); // 1,2 (transpose)
+               y(i12,qx,qy,e) = w_detJ * (-J21*R11 + J11*R21); // 2,1 (transpose)
```

Comment:
Was this a bug?

Reply:
Yes, I fixed multiple bugs like this in a previous PR. The unit tests used to be on meshes aligned with the coordinate axes, in which case the bug had no effect. I made the unit tests use more general meshes to expose bugs like this.